Cargo.toml

```diff
@@ -10,16 +10,16 @@ expose_stream = []
 [dependencies]
 bounded-static = "0.5.0"
 bytes = "1.5.0"
-imap-codec = { version = "2.0.0", features = ["quirk_crlf_relaxed", "bounded-static"] }
-imap-types = { version = "2.0.0" }
+imap-codec = { version = "2.0.0", features = ["quirk_crlf_relaxed", "bounded-static", "ext_id", "ext_sort_thread", "ext_binary", "ext_uidplus"] }
```

Comment:
Not sure if `imap-flow` should expose the same features as `imap-{types,codec}` or if it should just enable all of them.

Reply:
I have no opinion here. @duesee what do you think?

Reply:
No strong opinion either. Let's enable everything for now. Otherwise CI also needs to test some permutations...